backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -130,11 +131,49 @@ internal fun emitLLVM(context: Context) {
         irModule.acceptVoid(codegenVisitor)
     }
 
+    when (context.shouldEmbedBitcode()) {
```

Comment:
Is bitcode embedding performed before debug info generation intentionally?

Reply:
It is not